### PR TITLE
New stanford header and footer styling

### PR DESCRIPTION
**Pull request by caesar2164**
- some content changes as well

@stvstnfrd, @lydiali, @jen-nifer, & @mondiaz

Screenshots:

Before | After | Stanford.edu
------------ | ------------- | -------------
screen shot 2018-04-26 at 2 11 23 pm | screen shot 2018-04-26 at 3 16 54 pm | screen shot 2018-04-26 at 2 18 49 pm
screen shot 2018-04-26 at 2 11 33 pm | screen shot 2018-04-26 at 3 16 45 pm | screen shot 2018-04-26 at 2 19 01 pm
screen shot 2018-04-26 at 2 11 38 pm | screen shot 2018-04-26 at 3 16 38 pm | screen shot 2018-04-26 at 2 19 05 pm
screen shot 2018-04-26 at 2 11 48 pm | screen shot 2018-04-26 at 3 16 29 pm | screen shot 2018-04-26 at 2 19 34 pm
screen shot 2018-04-26 at 2 04 01 pm | screen shot 2018-04-26 at 3 15 38 pm | screen shot 2018-04-26 at 2 19 40 pm

### CODE REVIEW

File: lagunita/lms/templates/su-footer.html [hunk 1]
```diff
@@ -6,32 +6,36 @@
...
         </a>
       </div>
       <div id="su-footer-text" class="span10">
-        <ul>
-          <li class="home"><a href="http://www.stanford.edu">SU Home</a></li>
-          <li class="maps alt"><a href="http://visit.stanford.edu/plan/maps.html">Maps &amp; Directions</a></li>
-          <li class="search-stanford"><a href="http://www.stanford.edu/search/">Search Stanford</a></li>
-          <li class="terms alt"><a href="http://www.stanford.edu/site/terms.html">Terms of Use</a></li>
-          <li class="copyright"><a href="http://www.stanford.edu/site/copyright.html">Copyright Complaints</a></li>
+        <ul id="global-links">
```
Comment (stvstnfrd) [inline lagunita/lms/templates/su-footer.html:17]: Do these have to be IDs?
We can probably get away with `class`es..

File: lagunita/lms/static/css/vendor/su-identity.css [hunk 2]
```diff
@@ -1,291 +1,231 @@
+/*
```
Comment (stvstnfrd) [inline lagunita/lms/static/css/vendor/su-identity.css]: This comment doesn't add clarity; let's drop it.

And as a side note:
When code _does_ warrant comments, please don't use the long string of `=`s.
Those show up as false positives when grepping for conflict markers.

File: lagunita/lms/static/css/vendor/su-identity.css [hunk 3]
```diff
@@ -1,291 +1,231 @@
...
+========================================================
+Custom base styles
+*/
+
 #su-header .su-container,
 #su-footer .su-container {
   margin: 0 auto;
 }
-
```
Comment (stvstnfrd) [inline lagunita/lms/static/css/vendor/su-identity.css]: Generally speaking, please split out whitespace changes into a separate PR, or at the very least, a separate commit. It makes this code significantly more difficult to review, as close to 50% of the changes are cosmetic.

**Review by stvstnfrd — CHANGES_REQUESTED** [review]
My comments thus far are mostly superficial.
Can you remove the cosmetic changes (whitespace and comments) into a separate PR?
These unrelated changes make this more difficult to review.

**Comment by stvstnfrd** [comment]
@caesar2164 Can you edit your description to include a screenshot of the current footer on https://stanford.edu for easier side-by-side comparison?

File: lagunita/lms/templates/su-footer.html (hunk 1 above)
Comment (caesar2164) [reply lagunita/lms/templates/su-footer.html:17]: This is how stanford.edu does it ¯\_(ツ)_/¯

**Comment by mondiaz** [comment]
@caesar2164 Just want to make sure I'm catching everything I should be here in these before, after, and Stanford.edu screenshot comparisons.

- The subject of this PR mentions header and footer changes. But the screenshots just show footer changes, and I think that's all that Rob asked about. So are there any things actually changing in the header? Just looking for clarity. Either way is okay by me.

- The 15 screenshots appear to illustrate the way the footer changes based on window size. So the first screenshots show how things compare when at their narrowest (how they might appear on a smartphone), and the last screenshots show how things compare when at full screen. Is that correct?

- The footer links are taking up two lines instead of one, and there are a few links being added, namely:
     - Emergency Info
     - Privacy Policy
     - Trademark Notice

     Am I understanding that correctly?

- The copyright string at the bottom is very slightly different when I look at the after and stanford.edu screenshots. It looks like on stanford.edu, the bottom line changes a comma to a period after "Stanford University" ("©  Stanford University. Stanford, California 94305." instead of "© Stanford University, Stanford, California 94305"). Is that right?

Here's a screenshot of that minor difference in the copyright string at the bottom.

checking-that-i-understand-the-changes

I don't know if these things matter to Rob or others, but for my own sake, was just curious what's changing. This all looks fine to me, so will leave it to the others to actually yay / nay these changes.

**Comment by caesar2164** [comment]
Thanks for the request for better info @mondiaz!

- There are header changes as well, but they are just in code. (Nothing changes visually)
- Correct, I show a whole bunch of screenshots, because part of the changes is better media query styling. (for phones, tablets, etc...)
- Correct, there are now more links, including the specifically requested `Privacy Policy`.
- Good catch, I've now fixed it to match stanford.edu

**Comment by mondiaz** [comment]
@caesar2164 thanks! LGTM 👍

**Comment by stvstnfrd** [comment]
@caesar2164 Can you link to where these styles came from?

**Comment by caesar2164** [comment]
Sure @stvstnfrd, I got them from here:
https://identity.stanford.edu/web-mobile.html

And specifically from this file:
https://www-media.stanford.edu/su-identity/css/su-identity.css

**Comment by stvstnfrd** [comment]
@caesar2164 Can you add those links as comments somewhere relevant?

**Review by stvstnfrd — APPROVED** [review]
After you add comments linking to the documentation/references, +1

**Comment by stvstnfrd** [comment]
and squash commits, presumably..

**Pull request merged**